FlowCrypt/Controllers/Compose/ComposeViewController.swift

```diff
@@ -334,6 +334,16 @@ extension ComposeViewController {
             contextToSend.add(recipient: recipient)
             evaluate(recipient: recipient)
         }
+
+        input.quoteCCRecipients.forEach { email in
+            let recipient = ComposeMessageRecipient(email: email, type: .cc, state: decorator.recipientIdleState)
+            contextToSend.add(recipient: recipient)
+            evaluate(recipient: recipient)
+        }
```

Comment:
Looking at the code, please try the following scenario (in Gmail app for reference, and then also in FlowCrypt app)

1) send and email `to:A` `bcc:B`
2) open sent email. It should show `to:A` `bcc:B` (I think? since you are the sender, you should see this)
3) reply to that email

expected result (in Gmail - please confirm, in which case also in FlowCrypt): the reply will be initiated as `to:A`, `bcc:B`

Reply:
Well, IMO bcc should not appear in reply.
![image](https://user-images.githubusercontent.com/77344191/157427988-8a4b2475-c746-4107-850c-cb1a8ffebf66.png)

As you can see from below, `reply all` button is not even there when there are only bcc recipients 
![image](https://user-images.githubusercontent.com/77344191/157428066-11d061c7-8087-474f-911d-d7e36a336377.png)

Reply:
thank you